docker/base/Dockerfile

```diff
@@ -12,6 +12,7 @@ RUN apt-get update && apt-get install -y \
     libbz2-dev \
     libjpeg-dev \
     libpng-dev \
+    libxml2-dev \
```

Comment:
@mszabo-wikia - why do we need simplexml?

Reply:
Needed by Design System

Reply:
`SimpleXMLElement` right?

Reply:
Yup